conan/internal/api/new/bazel_lib.py

```diff
@@ -9,12 +9,12 @@
 class {{package_name}}Recipe(ConanFile):
     name = "{{name}}"
     version = "{{version}}"
-    package_type = "library"
+    package_type = "static-library"
```

Comment:
static one as Bazel, AFAIK, has no dynamic way to build a shared/static library. It has options as `--dynamic_mode=fully` but it's not working as expected. The only way that I got it was to add `linkshared=True` to the BUILD file in the test_package but it made useless the use of `shared=True` dynamically